lib/wings/model_transformer.rb

```diff
@@ -110,7 +110,7 @@ def fetch(key)
     private
 
     def mint_id
-      id = pcdm_object.assign_id
+      id = pcdm_object.try(:assign_id)
```

Comment:
`Hydra::AccessControl#assign_id` is a private method so just try here and don't assign an id.  Is this okay or should this be changed to `send(:assign_id)`?

Reply:
this seems good to me.